src/bin/compute-wer.cc

```diff
@@ -25,6 +25,41 @@
 #include "tree/context-dep.h"
 #include "util/edit-distance.h"
 
+
+
+namespace kaldi {
+
+
+template<typename T>
+void PrintAlignmentStats(const std::vector<T> &ref,
```

Comment:
I don't see this capability/function used in any external tool. Plus I'm sure we already do have this capability. So could you please remove this from this PR and eventually, if you think it's a "worthy" change, create a new PR, please?

Reply:
@jtrmal, could you please help us find the tool? Interesting, once I sent a PR to print similar data from the very same program at verbosity 2+, only in the raw form, and you then told me we already have that. I remember I could not find it.